### ISSUE TITLE
Making asking for help prohibited when someones question is not answered or can't be answered

### ISSUE THREAD

**Issue by Cyanitem** [plexidev/issue-tracker#8]
This gets me angry when people don't care about others problems and just send their message

**Comment by realCartar** [comment]
In my complete honest opinion,
I don't think this is a good idea. And here's why I think that: 
Due to the amount of newcomers that join the server and literally just join to ask a question - I highly doubt any of them would take the time to read any of the support rules (which they clearly don't), to find that you must wait for another person's question to be answered, before asking a question yourself. I just don't see it working due to the fact that people are impatient. 

I see where you're coming from, in the sense that it can get annoying when your question does get ignored, although there's not much we can do about it since the people that are kind enough to answer questions aren't paid or incentivised to do so.

I don't know how to phrase this to fully make sense, and not sound like an ass hole.

TL;DR - No.

**Comment by Revolx** [comment]
I understand where you are coming from with the annoyance of certain people not being able to get an answer and some users questions being pushed higher into the chat. Although it is something that does bug people, limiting new people to coding to not frankly allowing them to post assistance when they really dont understand coding themselves is not always the best approach. 

We want people to come to the server and use the support channels and not discourage them in any way.
This is not something that frankly is a big enough issue currently for us to look further into.

Nonetheless thank you for your opinion on the matter.
